### PR TITLE
Stage 0.3.1

### PR DESCRIPTION
**Pull request by cramertj**

### CODE REVIEW

File: futures-channel/Cargo.toml [hunk 1]
```diff
@@ -27,9 +27,5 @@ unstable = ["futures-core/unstable"]
 cfg-target-has-atomic = ["futures-core/cfg-target-has-atomic"]
 
 [dependencies]
-futures-core = { path = "../futures-core", version = "0.3.0", default-features = false }
-futures-sink = { path = "../futures-sink", version = "0.3.0", default-features = false, optional = true }
-
-[dev-dependencies]
-futures = { path = "../futures", version = "0.3.0", default-features = true }
-futures-test = { path = "../futures-test", version = "0.3.0", default-features = true }
```
Comment (Nemo157) [inline futures-channel/Cargo.toml]: Presumably you don't want to merge the dev-dependency removal?

Comment (Nemo157) [reply futures-channel/Cargo.toml]: Or are these no longer used?

Comment (cramertj) [reply futures-channel/Cargo.toml]: Heh, no, I don't, thanks for the catch.

**Comment by Nemo157** [comment]
LGTM with the dev-dependencies reinstated

**Pull request merged**